client/components/jetpack-product-card/features.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import { isArray, isObject } from 'lodash';
+import React, { useState, useCallback, FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ExternalLink from 'components/external-link';
+import FoldableCard from 'components/foldable-card';
+import FeaturesItem from './features-item';
+import { Features } from './types';
```

Comment:
We can use `import type` instead of `import` here.
https://www.typescriptlang.org/docs/handbook/modules.html#importing-types